### PR TITLE
feat: add language column to posts

### PR DESCRIPTION
**Pull request by omBratteng**

### CODE REVIEW

File: __tests__/workers/postUpdated.ts [hunk 1]
```diff
@@ -422,6 +422,34 @@ it('should save a new post with basic information', async () => {
...
+    id: 'f99a445f-e2fb-48e8-959c-e02a17f5e816',
+    title: 'Title',
+    url: 'https://post.com',
+    source_id: 'a',
+    order: 0,
+    language: 'nb',
+  });
+  const posts = await con.getRepository(Post).find();
+  expect(posts.length).toEqual(3);
+  expect(posts[2]).toMatchSnapshot({
```
Comment (rebelchris) [inline __tests__/workers/postUpdated.ts:440]: If we can lets remove snapshot and just make it inline evaluate.
We slowly want to get rid of them.

File: src/workers/postUpdated.ts [hunk 2]
```diff
@@ -506,6 +507,7 @@ const fixData = async ({
         ? markdown.render(data.extra.content)
         : undefined,
       videoId: data?.extra?.video_id,
+      language: data?.language,
```
Comment (rebelchris) [inline src/workers/postUpdated.ts:510]: Just check with @denisb0 if its under direct or extra prop?

**Review by rebelchris — COMMENTED** [review]
Minor comment on snapshot if possible and double check on attr.

File: src/workers/postUpdated.ts (hunk 2 above)
Comment (omBratteng) [reply src/workers/postUpdated.ts:510]: Direct. Already checked :)

File: __tests__/workers/postUpdated.ts (hunk 1 above)
Comment (omBratteng) [reply __tests__/workers/postUpdated.ts]: Removed snapshot

**Review by rebelchris — APPROVED** [review]
Nice! Thanks 🥳

File: __tests__/workers/postUpdated.ts [hunk 3]
```diff
@@ -422,6 +422,26 @@ it('should save a new post with basic information', async () => {
...
+    language: 'nb',
+  });
+  const posts = await con.getRepository(Post).find();
+  expect(posts.length).toEqual(3);
+  expect(posts[2]).toMatchObject({
+    sourceId: 'a',
+    title: 'Title',
+    showOnFeed: true,
+    language: 'nb',
   });
```
Comment (capJavert) [inline __tests__/workers/postUpdated.ts:445]: Might be flaky without sort.

**Review by capJavert — APPROVED** [review]

File: __tests__/workers/postUpdated.ts (hunk 3 above)
Comment (omBratteng) [reply __tests__/workers/postUpdated.ts:445]: I don't think `toMatchObject` does care about the order, as it only matches partials

Comment (omBratteng) [reply __tests__/workers/postUpdated.ts:445]: Took [the test from here](https://jestjs.io/docs/expect#tomatchobjectobject) and moved  the key `kitchen` in `desiredHouse`, and the test succeeded.

Comment (capJavert) [reply __tests__/workers/postUpdated.ts:445]: Not `toMatchObject` but `posts[2]` might not always be the same row since order is not guaranteed.

Comment (omBratteng) [reply __tests__/workers/postUpdated.ts:445]: Ah, yeah, that one 😬

**Pull request merged**